app/assets/stylesheets/amp_styles.scss

```diff
@@ -0,0 +1,166 @@
+$responsive: true;
+@import 'dough/assets/stylesheets/lib/variables';
+
+@font-face {
+  font-family: Museo Sans;
+  src: font-url('mas-fonts/app/assets/fonts/mas/fonts/MuseoSans_300.woff');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Museo Sans;
+  src: font-url('mas-fonts/app/assets/fonts/mas/fonts/MuseoSans_700.woff');
+  font-weight: 700;
+}
+
+// AMP specific classes
+.amp-article {
+  font-family: Museo Sans;
+  color: $color-grey-primary;
+  font-weight: 300;
+  line-height: 1.5rem;
+  font-size: 1rem;
+  -webkit-font-smoothing: antialiased;
+  padding: 0 0.6rem 1px 0.6rem;
+  background-color: white;
+  overflow: auto;
+}
+
+.amp-article__header {
+  background-color: $color-green-primary;
+  height: 2rem;
+  padding: 0.8rem 0 0.5rem 0.4rem;
+}
+
+.amp-article__mas-logo {
+  background-position: -12rem 0;
+  width: 12rem;
+  display: block;
+  background-image: url(/assets/logo-sprite-en.png);
+  height: 30px;
+  background-size: cover;
+}
+
+// AMP element and class styles (that come from contento)
+.amp-body {
+  max-width: 36rem;
+  margin: 0 auto;
+}
+
+body {
+  background-color: $color-grey-pale;
+}
+
+h1 {
+  font-size: 1.9rem;
+  line-height: 2.25rem;
+  margin-top: 0;
+  padding-top: 2rem;
+  margin-bottom: 1rem;
+  color: $color-green-secondary;
+}
+
+h2 {
+  font-size: 1.4rem;
+  line-height: 2rem;
+  margin-top: 1.5rem;
+  margin-bottom: 1rem;
+  color: $color-green-dark;
```

Comment:
These headers can all be the same colour (they are _very_ similar shades on the main site, and that's an oversight that we will correct)